clio-client/src/test/scala/org/broadinstitute/clio/client/dispatch/MoveGvcfSpec.scala

```diff
@@ -19,7 +19,8 @@ class MoveGvcfSpec extends BaseClientSpec {
     recoverToSucceededIf[IllegalArgumentException] {
       val command = MoveGvcf(
         key = testGvcfTransferV1Key,
-        destination = MockIoUtil.InvalidPath
+        destination = MockIoUtil.InvalidPath,
+        None
```

Comment:
could you use an overloaded constructor to avoid having to pass `None`?

Reply:
I can probably use a default value. Overloads don't play nicely with case classes AFAIK.